backend/tests/api/app_test.py

```diff
@@ -65,28 +65,7 @@ def test_chat_response_failure(mocker):
 
 @pytest.mark.asyncio
 async def test_lifespan_populates_db(mocker, mock_initial_data) -> None:

```

Comment:
IThe blob mocks in `mock_initial_data` can also now be removed and there's a test failure in this file that looks like a missing mock?